recipes-android/android/android-system-partition_skipjack-p.bb

```diff
@@ -1,12 +1,14 @@
 inherit gettext
 
-SUMMARY = "Downloads the Snapdragon Wear 2100/3100 /usr/libexec/hal-droid and /usr/include/android folders and installs them for libhybris"
+SUMMARY = "Downloads the TicWatch C2+ system folders and installs them for libhybris"
 LICENSE = "CLOSED"
 
-SRC_URI = "https://dl.dropboxusercontent.com/s/j05olh49xqcsdwl/system-sawfish-pie-preview.tar.gz"
+SRC_URI = "file://system.tar.gz"
```

Comment:
We should probably first push this to a Dropbox first before merging this :smile:

Reply:
I'm not sure how you want me to get it to you, don't wanna clutter the matrix room with 110 mb.

Reply:
Uploaded via dropbox, link at https://dl.dropboxusercontent.com/s/3scbpxkbxv5grtp/Skipjack-System.tar.gz.